src/Console/Console.ino

```diff
@@ -110,10 +112,11 @@ void loop() {
 
   // remote control.  There's a relay that will pull SYSTEM_ENABLE_PIN to LOW when pressed (enable gameplay).
   // goes to HIGH when pressed again (disable gameplay).
-  static boolean systemNormalMode = false;
+  static boolean systemNormalMode = true;
   if( systemEnable.update() ) {
     // system enable/disable state has changed.
-    systemNormalMode = systemEnable.read() == LOW;
+    // TODO: change this to LOW when it is properly wired up
+    systemNormalMode = systemEnable.read() == HIGH;
```

Comment:
Consider wrapping the fireEnable audible tones into a function, and then calling that function from here, too, with a different sets of audibles.  So, if the systemEnable state changes, we get audible tones, too.